_partials/keep_current.md

```diff
@@ -146,6 +146,37 @@ pyenv versions
   lewagon
 ```
 
+### Install the bootcamp packages
+
+```bash
+pip install -Ur <REQUIREMENTS_URL>
+pip install -U yapf jupyterlab seaborn plotly nbconvert xgboost statsmodels pandas-profiling dtale jupyter-resource-usage
```

Comment:
Would we add these dependencies in a <CERTIF_REQUIREMENTS_URL> or <COMPLETE_REQUIREMENTS_URL> ?

Reply:
good idea